.github/workflows/publish-release-artifact.yml

```diff
@@ -16,11 +16,11 @@ jobs:
       is-rc: ${{ steps.set-version.outputs.is-rc }}
     steps:
       - uses: actions/checkout@v5
-      - name: Set up JDK 17
+      - name: Set up JDK 21
```

Comment:
I don't think this action will work anymore since immutable releases were enabled?

<img width="734" height="98" alt="Image" src="https://github.com/user-attachments/assets/3c8e8f32-d0a6-4fdd-9290-ee64b912a033" />

Artifacts need to be attached at release time, so this needs to change to publish release drafter / an LTS or security template on tag, or jenkins-infra/release needs to take care of it.

(If we keep immutable release enabled, or this action should be migrated ahead of time)

Reply:
Interesting, that this happens after the release process. I've disabled it again.